spec/models/user_spec.rb

```diff
@@ -1,5 +1,11 @@
 require 'rails_helper'
 
-RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+describe User do
+	describe 'simple validations' do
+		it 'prevents users from having the same email' do
+			create :user, name: 'Bob Smith', email: 'bsmith@example.com'
```

Comment:
Since the names aren't relevant to this test, better to leave them out and let the factory handle them.